samples/snippets/csharp/VS_Snippets_ADO.NET/DP Custom CopyToDataTable Examples/CS/CustomCopyToDataTableCS.csproj

```diff
@@ -1,59 +1,22 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="3.5" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>9.0.20829</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{E2D91D7D-AA1D-416B-B399-704743A33293}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFramework>net48</TargetFramework>
+    <LangVersion>8.0</LangVersion>
```

Comment:
I'd prefer not setting C# 8 as the language version for a .NET Framework project. It isn't supported. Was there something in the code that required this?

Reply:
This sample is using the "coalescing assignment" language feature.

https://github.com/dotnet/docs/blob/f8887e0d68a67c07d91bcdc4845be70678bd815b/samples/snippets/csharp/VS_Snippets_ADO.NET/DP%20Custom%20CopyToDataTable%20Examples/CS/Program.cs#L320